source/server/http/admin.cc

```diff
@@ -492,6 +493,14 @@ Http::Code AdminImpl::handlerLogging(absl::string_view url, Http::HeaderMap&,
   return rc;
 }
 
+Http::Code AdminImpl::handlerMemory(absl::string_view, Http::HeaderMap&, Buffer::Instance& response,
+                                    AdminStream&) {
+  response.add(
```

Comment:
Can you use proto format for this? See https://github.com/envoyproxy/envoy/tree/master/api/envoy/admin/v2alpha. This the direction we want to move the API to make it more programmatic and expressable as proto/JSON/etc.

Reply:
Sure, I will do this instead.